Integrations/FortiGate/FortiGate.py

```diff
@@ -429,6 +467,76 @@ def create_firewall_service_request(service_name, tcp_range, udp_range):
     return response
 
 
+def ban_ip(ip_addresses_array, time_to_expire):
+    uri_suffix = 'monitor/user/banned/add_users/'
+
+    payload = {
+        'ip_addresses': ip_addresses_array
+    }
+    if time_to_expire:
+        time_to_expire_int = convert_arg_to_int(time_to_expire, "expiry")
+        payload["expiry"] = time_to_expire_int
+    response = http_request('POST', uri_suffix, {}, json.dumps(payload))
+    return response
+
+
+def ban_ip_command():
+    ip_addresses_string = demisto.args()["ip_addresses"]
+    ip_addresses_array = ip_addresses_string.split(",")
+    time_to_expire = demisto.args().get("expiry")
+    response = ban_ip(ip_addresses_array, time_to_expire)
+    demisto.results({
```

Comment:
use `return_outputs`.
add context output (you already set the output path in `get_banned_ips`

Reply:
AFAIK return_outputs doesn't handle well when you don't have any context outputs.
Moreover - we don't have context outputs by choice for these commands, it's a design choice verified with Arseny.

Reply:
ok